pallets/subtensor/src/staking.rs

```diff
@@ -343,7 +343,7 @@ impl<T: Config> Pallet<T> {
         Self::set_stakes_this_interval_for_coldkey_hotkey(
             &coldkey,
             &hotkey,
-            stakes_this_interval + 1,
+            stakes_this_interval.saturating_add(1),
```

Comment:
inconsistencies between using `saturating_inc` and `saturating_add` for increments of 1